Dockerfile

```diff
@@ -74,3 +75,36 @@ RUN	ln -s /opt/local/lib/livepeer-w3/livepeer-w3.js /usr/local/bin/livepeer-w3 &
 EXPOSE	1935	4242	8080	8889/udp
 
 CMD	["/usr/local/bin/MistController", "-c", "/etc/livepeer/catalyst.json"]
+
+FROM catalyst AS livepeer-in-a-box
+
+RUN	apt update && apt install -yqq \
+	rabbitmq-server \
+	nginx \
+	gdb \
+	inotify-tools \
+	file \
+	# for `shasum`
+	perl \
+	&& rm -rf /var/lib/apt/lists/*
+
+RUN curl -L -O https://binaries.cockroachdb.com/cockroach-v23.1.5.linux-amd64.tgz \
+	&& tar xzvf cockroach-v23.1.5.linux-amd64.tgz \
+	&& mv cockroach-v23.1.5.linux-amd64/cockroach /usr/bin/cockroach \
+	&& rm -rf cockroach-v23.1.5.linux-amd64.tgz cockroach-v23.1.5.linux-amd64
+
+RUN curl -o /usr/bin/minio https://dl.min.io/server/minio/release/linux-amd64/minio \
+	&& curl -o /usr/bin/mc https://dl.min.io/client/mc/release/linux-amd64/mc \
+	&& chmod +x /usr/bin/minio /usr/bin/mc
+
+ADD ./scripts /usr/local/bin
+ADD ./config/full-stack.json /config/full-stack.json
```

Comment:
would recommend using `COPY --link`: https://docs.docker.com/engine/reference/builder/#copy---link

Reply:
should the config go in `/etc/livepeer/`?